src/db_impl_gc.cc

```diff
@@ -14,9 +14,8 @@ void TitanDBImpl::MaybeScheduleGC() {
 
   if (shuting_down_.load(std::memory_order_acquire)) return;
 
-  while (unscheduled_gc_ > 0 &&
+  while (!gc_queue_.empty() &&
```

Comment:
`gc_queue_` would not pop immediately, this leads to schedule much more than `gc_queue_.size()`

Reply:
so it's necessary for the unscheduled_gc, and that is why rocksdb define unscheduled_flush and unscheduled_compaction...

Reply:
```c++
auto i = std::min((int32_t)gc_queue_.size(), db_options_.max_background_gc - 
 bg_gc_scheduled_);
while (i--) {
```
is not enough?

Reply:
so...why not using unscheduled_gc_, it is more comprehensible

Reply:
alright, keep the `unscheduled_gc_`